docs/file-reference.md

```diff
@@ -54,6 +56,13 @@ Each service has name and list of the containers.
 
 Name of the service.
 
+#### replicas
+| type  | required |
+|-------|----------|
+| int32 |    no    |
```

Comment:
I think `int32` in the code is ok.

I'm just thinking if it makes sense in documentation mention `int32` type, maybe just `integer`?